package.json

```diff
@@ -33,4 +33,4 @@
     "hexo-server": "^0.2.0",
     "replacestream": "^4.0.0"
   }
-}
+}
```

Comment:
Can you revert this one? After running `npm install`, this gets reverted.

Reply:
Done. Thanks.